app/controllers/admin/works_controller.rb

```diff
@@ -126,6 +126,13 @@ def download_ohms_xml
   # PATCH/PUT /admin/works/ab2323ac/create_combined_audio_derivatives
   # Unfortunately, if the job fails for any reason, the user will not be notified.
   def create_combined_audio_derivatives
+    if CombinedAudioDerivativeCreator.new(@work).audio_members.empty?
```

Comment:
Small thing, but to be a little bit more "law of demeter" encapsulated, let's make a method for this on `CombinedAudioDerivativeCreator`.  Like `if CombinedAudioDerivativeCreator.new(@work).no_input_available?` or `~ CombinedAudioDerivativeCreator.new(@work).available_members?`. 

Then you can use the same method *internal* to `CombinedAudioDerivativeCreator` to be sure that we're using the same logic here to decide to error redirect that the `CombinedAudioDerivativeCreator` is using to decide if it can do it's thing.

Reply:
Good call.